server/persistence/dynamo/dynamo_service_interaction.go

```diff
@@ -0,0 +1,257 @@
+//nolint:dupl
```

Comment:
is this a nolint for the whole file? That's kinda gross -- why do we want to ignore it?

Reply:
yea; i copied this from mongo. :crossed_fingers: that i can simply remove it